code/game/atoms.dm

```diff
@@ -15,6 +15,10 @@
 	var/atom_say_verb = "says"
 	var/dont_save = 0 // For atoms that are temporary by necessity - like lighting overlays
 
+	// A list of datums that desire to interact with this atom - used so that automated processes
+	// don't step on one another's toes, if they are doing an action that takes time
+	var/list/interaction_queue = list()

```

Comment:
For the love of god, **don't** add another list creation to the initialization step of every atom in the entire game just to make competing autogibbers work more reliably.